x-pack/plugins/event_log/server/lib/ready_signal.test.ts

```diff
@@ -13,28 +13,9 @@ describe('ReadySignal', () => {
     readySignal = createReadySignal<number>();
   });
 
-  test('works as expected', async (done) => {
-    let value = 41;
-
-    timeoutSet(100, async () => {
-      expect(value).toBe(41);
-    });
-
-    timeoutSet(250, async () => readySignal.signal(42));
-
-    timeoutSet(400, async () => {
-      expect(value).toBe(42);
-
-      const innerValue = await readySignal.wait();
-      expect(innerValue).toBe(42);
-      done();
-    });
-
-    value = await readySignal.wait();
-    expect(value).toBe(42);
+  test('works as expected', async () => {
```

Comment:
@pmuellr, it seems this covers the ReadySignal class - but would you mind confirming?